MAUI/Rating/Getting-Started.md

```diff
@@ -0,0 +1,189 @@
+
+## Adding a .NET MAUI Rating reference
+
+Syncfusion .NET MAUI controls are available in [Nuget.org](https://www.nuget.org/). To add [.NET MAUI Rating] to your project, open the NuGet package manager in Visual Studio, search for [Syncfusion.Maui.Inputs](https://www.nuget.org/packages/Syncfusion.Maui.Inputs) and then install it.
+
+## Handler registration 
+
+In the MauiProgram.cs file, register the handler for Syncfusion core.
+
+{% highlight c# hl_lines="6 7 18 19" %}
+using Microsoft.Maui;
+using Microsoft.Maui.Hosting;
+using Microsoft.Maui.Controls.Compatibility;
+using Microsoft.Maui.Controls.Hosting;
+using Microsoft.Maui.Controls.Xaml;
+using Syncfusion.Maui.Core.Hosting;
+using Syncfusion.Maui.ListView.Hosting;
+
+namespace RatingSample
+{
+    public static class MauiProgram
+    {
+        public static MauiApp CreateMauiApp()
+        {
+            var builder = MauiApp.CreateBuilder();
+            builder
+            .UseMauiApp<App>()
+            .ConfigureSyncfusionCore()
+            .ConfigureSyncfusionListView()
+            .ConfigureFonts(fonts =>
+            {
+                fonts.AddFont("OpenSans-Regular.ttf", "OpenSansRegular");
+            });
+
+            return builder.Build();
+        }      
+    }
+}   
+
+{% endhighlight %} 
+
+## Adding the .NET MAUI Rating control
+
+Step 1: Add the NuGet to the project as discussed in the above reference section. 
+
+Step 2: Add the namespace as shown in the following code sample:
+
+{% tabs %}
+
+{% highlight xaml %}
+
+	<xmlns:rating="clr-namespace:Syncfusion.Maui.Inputs;assembly=Syncfusion.Maui.Inputs"/>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+	using Syncfusion.Maui.Inputs;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Initialize Rating
+
+Now, add the [`SfRating`] control with a required optimal name using the included namespace.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+	<rating:SfRating x:Name="rating" />
+	
+{% endhighlight %}
+
+{% highlight C# %}
+
+SfRating rating;
+public MainPage()
+{
+    InitializeComponent();
+    rating = new SfRating();
+    this.Content = rating;
+} 
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Set Number of Rating Items
+
+The number of rating items to be displayed can be customized in the [`SfRating`] control. Users can create a rating application with 5 items as follows. The [`ItemCount`] property is used to define the number of rating items.
+
+N> The default value of [`ItemCount`] is 5.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+	<rating:SfRating ItemCount="5" />
+	
+{% endhighlight %}
+
+{% highlight C# %}
+
+SfRating rating;
+public MainPage()
+{
+    InitializeComponent();
+    rating = new SfRating();
+    rating.ItemCount = 5;
+}
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Set Value
+
+The display value can be set in the [`SfRating`] control, which is selected among the items. The following code example shows the display value of 3 with 5 rating items. The [`Value`] property is used to set display value.
```

Comment:
The display value can be set in the [`SfRating`] control, which is selected among the items. The following code example shows the display value of three with five rating items. The [`Value`] property is used to set the display value.

Reply:
Changed